cg/cli/clean.py

```diff
@@ -253,11 +255,11 @@ def hk_bundle_files(
     LOG.info(f"Process freed {round(size_cleaned * 0.0000000001, 2)}GB. Dry run: {dry_run}")
 
 
-@clean.command("flow-cells")
+@clean.command("illumina-sequencing-runs")
 @DRY_RUN
 @click.pass_obj
-def clean_flow_cells(context: CGConfig, dry_run: bool):
-    """Remove flow cells from the flow cells and demultiplexed runs folder."""
+def clean_illumina_sequencing_runs(context: CGConfig, dry_run: bool):
+    """Remove raw and demultiplexed illumina sequencing runs from the from hasta."""
```

Comment:
I think we should stick to the agreed jargon of `sequencing` and `demultiplex`, `raw` may be unclear
```suggestion
    """Remove Illumina run sequencing and demultiplex data from the from hasta."""
```